.github/workflow_scripts/e2e_mgpu_check.sh

```diff
@@ -1,6 +1,8 @@
 # Move to parent directory
 cd ../../
 
+pip install install --extra-index-url https://pypi.nvidia.com pylibwholegraph-cu11
```

Comment:
Is this a typo here?
```suggestion
pip install --extra-index-url https://pypi.nvidia.com pylibwholegraph-cu11
```